handlers/apps.go

```diff
@@ -17,6 +17,7 @@ type appListParams struct {
 	//
 	// in: path
 	Namespace string `json:"namespace"`
+	AppName   string `json:"app"`
```

Comment:
And same comment here. I think I prefer having a type for `appListParams` and `appDetailsParams`.

Here, I think this introduces a weird user experience, because this will land in the Api docs will which will show `AppName` as a parameter for the AppList endpoint.

Reply:
I agree with that...we should create a common type params for all they are so similar but we have not time now we can do it later. Let me create a new type tomorrow